tests/e2e/flowtest/mcg/test_mcg_namespace_disruptions.py

```diff
@@ -0,0 +1,306 @@
+import json
+import logging
+import uuid
+
+import pytest
+import botocore.exceptions as boto3exception
+
+from ocs_ci.framework.pytest_customization.marks import skipif_aws_creds_are_missing, flowtests
+from ocs_ci.framework.testlib import E2ETest, skipif_ocs_version
+from ocs_ci.ocs.bucket_utils import (
+    sync_object_directory, put_bucket_policy,
+    get_bucket_policy, s3_put_object,
+    s3_get_object, namespace_bucket_update,
+    rm_object_recursive, verify_s3_object_integrity
+)
+from ocs_ci.framework import config
+from ocs_ci.ocs import constants, defaults
+from ocs_ci.ocs.exceptions import UnexpectedBehaviour
+from ocs_ci.ocs.node import drain_nodes, wait_for_nodes_status, schedule_nodes
+from ocs_ci.ocs.resources.bucket_policy import NoobaaAccount, gen_bucket_policy, HttpResponseParser
+from ocs_ci.ocs.resources import pod
+from ocs_ci.helpers.helpers import wait_for_resource_state
+
+logger = logging.getLogger(__name__)
+
+MCG_NS_RESULT_DIR = '/result'
+MCG_NS_ORIGINAL_DIR = '/original'
+
+
+def setup_base_objects(awscli_pod, amount=2):
+    """
+    Prepares two directories and populate one of them with objects
+
+     Args:
+        awscli_pod (Pod): A pod running the AWS CLI tools
+        amount (Int): Number of test objects to create
+
+    """
+    awscli_pod.exec_cmd_on_pod(command=f'mkdir {MCG_NS_ORIGINAL_DIR} {MCG_NS_RESULT_DIR}')
+
+    for i in range(amount):
+        object_key = f"ObjKey-{i}"
+        awscli_pod.exec_cmd_on_pod(
+            f"dd if=/dev/urandom of={MCG_NS_ORIGINAL_DIR}/{object_key} bs=1M count=1 status=none"
+        )
```

Comment:
you are using the same code over here as well,
https://github.com/red-hat-storage/ocs-ci/blob/master/tests/e2e/lifecycle/mcg/test_mcg_namespace_lifecyle.py#L27-L42
can we create a common function?

Reply:
ack